install_config/aggregate_logging.adoc

```diff
@@ -855,7 +855,18 @@ spec:
 ----
 <1> This label should uniquely identify a replica with a single node that bears that
 label, in this case `logging-es-node=1`. Use the `oc label` command to apply
-labels to nodes as needed.
+labels to as many nodes as needed.
+
+For example, if your deployment has three infrastructures, you could add labels for those
```

Comment:
I think this needs to be two steps - one to create the nodeSelector and another to apply it to your nodes.

Do you mean "infrastructure nodes" instead of "infrastructures?"

Reply:
Thanks, changed to 2 steps and corrected terminology.